src/error.rs

```diff
@@ -43,13 +50,15 @@ impl error::Error for SdpParserError {
             SdpParserError::Unsupported { ref message, .. } |
             SdpParserError::Sequence { ref message, .. } => message,
             SdpParserError::Integer(ref err) => error::Error::description(err),
+            SdpParserError::Address(ref err) => error::Error::description(err),
```

Comment:
Why not use `err.description()`? I suppose this goes for the line above as well.